server/src/__tests__/server.test.ts

```diff
@@ -222,7 +222,7 @@ describe('server', () => {
         {} as any,
       )
 
-      expect(result).toHaveLength(1)
+      expect(result).toHaveLength(3)
```

Comment:
Let us check what all the code actions are about. Consider just doing a snapshot test of the entire result object.